.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         node-version:
-          - 18
+          - 22
```

Comment:
Should this have be updated too?
https://github.com/sindresorhus/ky/blob/93c02ac7817df1138e97e8971fe5165e30d68097/package.json#L22

Reply:
No. But I should have added 22, instead of replacing 18 here.

We were experiencing some strange CI failures that I couldn't replicate locally, so I tried updating Node here to see if that had any effect, which it didn't. In the end, it seems that GitHub Actions was just having a bad day, as it resolved itself. I didn't think about adding 18 back to the testing matrix, but that would be a good idea.

Nothing has changed in Ky to require users update Node. However, Node 18 is reaching end of life at the end of this month. So we won't be supporting it soon, anyway.